src/workerd/io/hibernation-manager.h

```diff
@@ -187,6 +187,11 @@ class HibernationManagerImpl final : public Worker::Actor::HibernationManager {
     TagCollection(TagCollection&& other) = default;
   };
 
+  struct AutoRequestResponsePair {
+    kj::StringPtr request;
```

Comment:
It might just be paranoia on my end but I'd say it's worth making these as `kj::String` and not worrying about lifetime for `StringPtr`.